TESTS/mbedmicro-rtos-mbed/rtostimer/main.cpp

```diff
@@ -347,16 +330,15 @@ utest::v1::status_t test_setup(const size_t number_of_cases)
 }
 
 Case cases[] = {
-    Case("Test one-shot not restarted when elapsed", test_oneshot_not_restarted),
-    Case("Test periodic repeats continuously", test_periodic_repeats),
-    Case("Test timer can be restarted while running", test_restart),
-    Case("Test stopped timer can be started again", test_start_again),
-    Case("Test restart changes timeout", test_restart_updates_delay),
-    Case("Test can be stopped", test_stop),
-    Case("Test timer is created in stopped state", test_created_stopped),
-    Case("Test timer started with infinite delay", test_wait_forever),
-    Case("Test timer started with zero delay", test_no_wait),
-    Case("Test calls from ISR fail", test_isr_calls_fail)
+    Case("One-shot not restarted when elapsed", test_oneshot_not_restarted),
+    Case("Periodic repeats continuously", test_periodic_repeats),
+    Case("Stopped timer can be started again", test_start_again),
```

Comment:
Why are we removing one of the tests?

Reply:
Sorry, I should have mentioned that. I decided to remove `test_restart` because it basically duplicates `test_restart_updates_delay` with a different delay.